syntax = "proto3";

package djls.v1.check;

message HealthRequest {}
message HealthResponse {
    bool passed = 1;
    optional string error = 2;
}

message GeoDjangoPrereqsRequest {}
message GeoDjangoPrereqsResponse {
    bool passed = 1;
    optional string error = 2;
}
